src/Microsoft.DotNet.Build.Tasks/PackageFiles/tool-runtime/nuget.config

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I kind of don't like having his extra nuget.config, so I think we should consider using the --source option.